src/app/components/mnoe-users-local-list/mnoe-users-local-list.coffee

```diff
@@ -78,6 +78,66 @@
           MnoErrorsHandler.processServerError(error)
       ).finally(-> user.isSendingInvite = false)
 
+    scope.availableRoles = () ->
+      [
+        "mnoe_admin_panel.constants.user_roles.member",
+        "mnoe_admin_panel.constants.user_roles.admin",
+        "mnoe_admin_panel.constants.user_roles.super_admin"
+      ]
+
+    scope.indexOfRoleInTable = (role) ->
+      scope.availableRoles().indexOf(scope.tlKeyFromUserRole(role))
+
+    # role here should always be in English (as returned by MnoHub)
+    scope.tlKeyFromUserRole = (role) ->
+      enTranslationTable = $translate.getTranslationTable('en-AU')
+      _.findKey(enTranslationTable, _.partial(_.isEqual, role))
```

Comment:
Would not be more readable extract `_.partial(_.isEqual, role)` and add a comment the same way you did with `$translate.getTranslationTable('en-AU')`.?